src/rgw/setup/rgw.py

```diff
@@ -92,6 +94,7 @@ def config(conf: MappedConf):
         """Performs configurations."""
 
         Log.info('Config phase started.')
+        Rgw._is_consul_alive(conf)
```

Comment:
function name is misleading. we can rename it to _check_consul_connection()

Reply:
done